### PR TITLE
PE-974 - Add support to CCX courses.

### PR DESCRIPTION
**Pull request by Squirrel18**
## Description:

This PR adds support for CCX course ids.
Also, it fixes a problem with the tracking log schema.

## Reviewers:

- [ ] @andrey-canon 
- [ ] @diegomillan

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -45,6 +45,7 @@ def findfiles(pat):
         'edx-opaque-keys==1.0.1',
         'pandas==0.24.2',
         'Unidecode==1.1.1',
+        'edx-ccx-keys==0.2.2',
```
Comment (andrey-canon) [inline setup.py:48]: why is this necessary ?

Comment (Squirrel18) [reply setup.py:48]: Opaque keys require it to recognize CCX keys, otherwise it will raise an InvalidKeyError. @andrey-canon

**Review by diegomillan — APPROVED** [review]
